app/controllers/borrow_requests_controller.rb

```diff
@@ -1,5 +1,11 @@
 class BorrowRequestsController < ApplicationController
-  before_action :check_logged_in?, only: %i(index show)
+  MAX_ALLOWED_BORROW_REQUESTS = Settings.borrow_request.max_allowed.freeze
+
+  before_action :check_logged_in?, only: %i(index show new create cancel)
```

Comment:
```suggestion
  before_action :check_logged_in?
```
Chị nghĩ mún vào controller này bắt buộc user phải login nên em ko cần cái `only` mô

Reply:
Đã fixed ạ